examples/configuration.yaml

```
@@ -3,6 +3,6 @@ kind: Configuration
 metadata:
   name: platform-ref-aws
 spec:
-  package: xpkg.upbound.io/upbound/platform-ref-aws:v0.5.0
+  package: xpkg.upbound.io/upbound/platform-ref-aws:v0.5.1
```

```suggestion
  package: xpkg.upbound.io/upbound/platform-ref-aws:v0.6.0
```
this change is good for `minor` according to https://semver.org/